.github/workflows/update-credits.yml

```diff
@@ -4,51 +4,36 @@ on:
   workflow_dispatch:
   schedule:
     - cron: 0 0 * * 0
-    
+
 jobs:
   get_credits:
     runs-on: ubuntu-latest
     # Hey there fork dev! If you like to include your own contributors in this then you can probably just change this to your own repo
     # Do this in dump_github_contributors.ps1 too into your own repo
-    if: github.repository == 'space-wizards/space-station-14'
-    
+    if: github.repository == 'new-frontiers-14/frontier-station-14'
+
     steps:
       - uses: actions/checkout@v3.6.0
         with:
           ref: master
-        
+
       - name: Get this week's Contributors
         shell: pwsh
-        env:
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
         run: Tools/dump_github_contributors.ps1 > Resources/Credits/GitHub.txt
```

Comment:
This should probably be in its own file.

Reply:
Do what you need man of stone